build/build.sh

```diff
@@ -31,6 +31,8 @@ if [ -z "${VERSION}" ]; then
     exit 1
 fi
 
+cat scripts/install-cni.sh | sed "s/@VERSION@/${VERSION}/" > scripts/install-cni-build.sh
```

Comment:
Suggest removing the unnecessary use of cat:
sed -e "s/@VERSION@/${VERSION}/g" scripts/install-cni.sh > scripts/install-cni-build.sh
Also consider if we want to surround the substitution string with @ or just at the beginning. I'm okay with either one, but let's be consistent.

Reply:
Oh, it got merged before I update it, will do it in the next pr.
Thanks.